src/concall_tools.py

```diff
@@ -25,8 +28,19 @@ def get_speakers(pdf_name, algorithm="auto"):
 
     doc = fitz.open(pdf_name)
     speakers = []
+    names=[]
+    speakers2=[]
     if algorithm == "auto" or algorithm == "bold":
         speakers = _get_speakers_in_bold(doc)
     if not speakers and (algorithm == "auto" or algorithm == "plain"):
         speakers = _get_speakers_from_text(doc)
-    return speakers
+    speakers1=_get_speakers_capitals(pdf_name)
+    for s in speakers:
+        names.append(s[0])
+    for s in speakers1:
```

Comment:
We can have another algo type as capitals and maybe just return the results from that algo only.

Reply:
After getting speaker names and firms from the capitals method, I'm finding common names and returning them to reduce the possibility of false positives